hbase-server/src/test/java/org/apache/hadoop/hbase/StartTestingClusterOption.java

```diff
@@ -171,13 +182,22 @@ public boolean isCreateWALDir() {
     return createWALDir;
   }
 
+  public boolean isExternalDFS() {
```

Comment:
I prefer here we pass in the necessary configs to connecting an external HDFS, and set them into the Configuration object.

Reply:
It would be complicated to pass all the required configuration in kv mode, especially HA or Kerberos, I want to pass an HDFS conf and add it to this.conf, is that ok?